nbs/docs/Prediction_intervals_in_forecasting_models.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<p>The title here is still "Installing Statsforecast"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/mlforecast/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>